### PR TITLE
feat(carousel): add config service to provide default values

### PR DESCRIPTION
**Pull request by jnizet**
refs #518

### CODE REVIEW

File: demo/src/app/components/carousel/demos/config/carousel-config.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<ngb-carousel>
+  <template ngbSlide>
+    <img src="http://lorempixel.com/900/500?r=4">
```
Comment (wesleycho) [inline demo/src/app/components/carousel/demos/config/carousel-config.html]: All `img` elements should have an `alt` attribute

**Comment by icfantv** [comment]
@jnizet please rebase.  thanks.

**Comment by jnizet** [comment]
@icfantv rebase done.

**Comment by icfantv** [comment]
LGTM.

**Pull request closed without merging**